website/pages/en/sunrise.mdx

```diff
@@ -38,15 +38,15 @@ To upgrade a hosted service subgraph, you can visit the subgraph dashboard on th
 2. Select the receiving wallet (the wallet that will become the owner of the subgraph).
 3. Click the "Upgrade" button.
 
-Once your subgraph is published, the [upgrade Indexer](#what-is-an-upgrade-indexer) will begin serving queries on it, so you can begin making queries immediately, once you have generated an API key ([learn more](/cookbook/upgrading-a-subgraph/#what-next)).
+Once your subgraph is published, the [Upgrade Indexer](#what-is-the-upgrade-indexer) will begin serving queries on it, so you can begin making queries immediately, once you have generated an API key ([learn more](/cookbook/upgrading-a-subgraph/#what-next)).
```

Comment:
No, "upgrade" [shouldn't be capitalized](https://www.notion.so/edgeandnode/The-Graph-Style-Guide-70a1409024d84bfbb4eccdf0e49b7aa7?pvs=4#09d4ed7a5a5d423aa1074e7ff58e53c1):
```suggestion
Once your subgraph is published, the [upgrade Indexer](#what-is-the-upgrade-indexer) will begin serving queries on it, so you can begin making queries immediately, once you have generated an API key ([learn more](/cookbook/upgrading-a-subgraph/#what-next)).
```